packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -550,7 +646,7 @@ export class SideTabBar extends ScrollableTabBar {
             } else {
                 rd = { title, current, zIndex };
             }
```

Comment:
variable rd stands for ?, may be a better name would be nice

Reply:
`rd` (`SideBarRenderData`) is not part of the changes from @fangnx, therefore I believe it should not be refactored unnecessarily in the PR at the moment.

Reply:
OK